WebUI/Data/Models/ApplicationUser.cs

```diff
@@ -30,9 +30,6 @@ public ApplicationUser()
         [StringLength(50)]
         public string LastName { get; set; }
 
-        [Column("birthday")]
-        public DateTime BirthDay { get; set; }
-
```

Comment:
Why are we removing the birthday from the user?

Reply:
When I initially pulled from master when I first made the branch it had birthday in there twice

Reply:
Right, makes sense - that's a known issue, just make sure it's not lost